src/cell.rs

```diff
@@ -22,8 +22,8 @@ pub struct Cell {
 impl Cell {
     /// Create a new `Cell` initialized with content from `string`.
     /// Text alignment in cell is configurable with the `align` argument
-    pub fn new_align(string: &str, align: Alignment) -> Cell {
-        let content: Vec<String> = string.lines().map(|x| x.to_string()).collect();
+    pub fn new_align<S: AsRef<str>>(string: S, align: Alignment) -> Cell {
```

Comment:
`AsRef` vs `ToString` ... I never know which one would be better.
I guess in that case `AsRef` will indeed be better, and probably backward compatible